src/main/java/uk/gov/pay/api/resources/HealthCheckResource.java

```diff
@@ -0,0 +1,40 @@
+package uk.gov.pay.api.resources;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.dropwizard.setup.Environment;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.Response;
+
+import static javax.ws.rs.core.MediaType.APPLICATION_JSON;
+
+@Path("/")
+public class HealthCheckResource {
+    public static final String HEALTHCHECK = "healthcheck";
+
+    @JsonProperty
+    private Boolean healthy;
+
+    @JsonIgnore
+    Environment environment;
+
+    public HealthCheckResource(Environment environment) {
+        this.environment = environment;
+    }
+
+    @GET
+    @Path(HEALTHCHECK)
+    @Produces(APPLICATION_JSON)
+    public Response captureCharge(@PathParam("chargeId") String chargeId) {
+        healthy = environment.healthChecks().getNames().size() == environment.healthChecks().runHealthChecks()
+                .values()
+                .stream()
+                .filter(healthcheck -> healthcheck.isHealthy() == true)
```

Comment:
we could possibly simplify this by using a method reference like .filter(HealthCheck.Result::isHealthy)